### DISCUSSION TITLE
Help please!!! Search is not working

### DISCUSSION THREAD

**Discussion by egwrk** [microsoft-search/pnp-modern-search#3776] [Q&A] [answered]
I have a page with all 3 pnp web parts configured but the search isn't working. When searching for 'Aviation', which is in the Additional Information column, it doesn't work. However, if searching for 'Aviation ' (Aviation with a space), it shows results. Can someone please explain why it's behaving this way and how to fix it? The Additional Information column is a multi-lines of text column so not sure if that might be contributing to the issue. Rich-text is also disabled though it was enabled previously. 

The stakeholders will review the page this Wednesday (June 5) and we have communications going out this Thursday (June 6) 0900 ET linking to the page. I would greatly appreciate any assistance!

**Without space:**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/56ac8807-f7ad-408b-bb83-927183692d84)

**With space:** 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/161059375/ad2b7cd2-a1b8-491b-9e0e-9e23090b7180)

**Below are the search queries I've tried but the issue still persists:** 
```
{searchTerms} AND
Path:"https://********/Lists/Learning" AND -Path:"https://********/Lists/Learning/AllItems.aspx"
```

```
(({searchTerms} OR RefinableString121:{searchTerms} OR RefinableString122:{searchTerms} OR RefinableString123:{searchTerms} OR RefinableString124:{searchTerms} OR RefinableString125:{searchTerms} OR RefinableString126:{searchTerms} OR RefinableString127:{searchTerms} OR RefinableString128:{searchTerms} OR RefinableString129:{searchTerms} OR RefinableString130:{searchTerms}) AND
Path:"https://********/Lists/Learning" AND -Path:"https://********/Lists/Learning/AllItems.aspx"")
```

**Comment by BVSanjay** [comment]
Try the below query
{searchTerms}* AND
Path:"https://********/Lists/Learning" AND -Path:"https://********/Lists/Learning/AllItems.aspx"

**Reply by wobba** [reply]
Here's my proposed query template for returning list items only, and if you have mapped the cp for the column to RefinableString122 you should have a read on my blog posts https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html and https://www.techmikael.com/2014/12/solution-to-cannot-search-content-in.html.

`({queryTerms}*) Path:"https://********/Lists/Learning" contentclass:sts_listitem_*`

You may also use `ListId:<id>` instead of path.

**Reply by egwrk** [reply]
Thanks so much for your responses! The issue seemed to have resolved itself, we checked the page this morning and it's now returning the expected results.

**Reply by egwrk** [reply]
> Here's my proposed query template for returning list items only, and if you have mapped the cp for the column to RefinableString122 you should have a read on my blog posts https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html and https://www.techmikael.com/2014/12/solution-to-cannot-search-content-in.html.
> 
> `({queryTerms}*) Path:"https://********/Lists/Learning" contentclass:sts_listitem_*`
> 
> You may also use `ListId:<id>` instead of path.

Sorry, what do you mean by '"cp"?

**Reply by kasperbolarsen** [reply]
Crawled property

**Comment by egwrk** [comment, marked as answer]
>Thanks so much for your responses! The issue seemed to have resolved itself, we checked the page this morning and it's now returning the expected results.

I had another PnP that had the same issue except this time the issue did not resolve itself. I updated the query text to the below and it has been working so far:

`(((*{searchTerms}*) OR RefinableString112:(*{searchTerms}*) OR RefinableString113:(*{searchTerms}*) OR RefinableString114:(*{searchTerms}*) OR RefinableString115:(*{searchTerms}*)) AND Path:"https://xxxxx/sites/xxxxx/Lists/xxxxx" AND Size >= 0)`

**Reply by wobba** [reply]
@egwrk Your query template is incorrect. Search do not support pre-fix wildcard so adding `*` before `{searchTerms}` will not work. I suggest you use this template which is syntactically correct: 

 ```
 ((({searchTerms}*) OR RefinableString112:({searchTerms}*) OR RefinableString113:({searchTerms}*) OR RefinableString114:({searchTerms}*) OR RefinableString115:({searchTerms}*)) AND Path:"https://xxxxx/sites/xxxxx/Lists/xxxxx" AND Size >= 0)
 ```
 
 And I assume you have the first part without a property filter due to this side-effect? https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html
